handler/core/src/main/java/io/knotx/fragments/handler/action/cb/CircuitBreakerAction.java

```diff
@@ -44,13 +46,25 @@
   private final Action doAction;
   private final ActionLogLevel actionLogLevel;
   private final String alias;
+  private final Set<String> errorTransitions;
 
   CircuitBreakerAction(CircuitBreaker circuitBreaker, Action doAction, String alias,
```

Comment:
I would remove this constructor. Instead, in `CircuitBreakerActionFactoryOptions` class you can initialise errorTransitions with ERROR_TRANSITION (please check that converter does not override this value).